src/command_modules/azure-cli-acr/README.rst

```diff
@@ -9,54 +9,55 @@ Commands to manage Azure container registries
         az acr: Commands to manage Azure container registries.
 
     Subgroups:
-        credential: Manage admin user credential for Azure container registries.
+        credential: Manage administrator login credentials for Azure container registries.
         repository: Manage repositories for Azure container registries.
 
     Commands:
-        check-name: Check whether the container registry name is available.
-        create    : Create a container registry.
-        delete    : Delete a container registry.
-        list      : List container registries.
-        show      : Get a container registry.
-        update    : Update a container registry.
+        check-name: Checks whether the container registry name is available for use.
+        create    : Creates or updates a container registry with the specified parameters.
```

Comment:
It doesn't update so much as overwrite. Every create in the CLI that is backed by a PUT does the same thing, but we don't advertise that because it will confuse and/or frustrate users. Invariably someone will just use 'create' to update something and then be suprised when some optional parameter was thrown away instead of being left alone.